cfgov/cfgov/settings/base.py

```diff
@@ -479,18 +479,26 @@
 REGSGOV_BASE_URL = os.environ.get('REGSGOV_BASE_URL')
 REGSGOV_API_KEY = os.environ.get('REGSGOV_API_KEY')
 
-# Akamai
+# CDNs
+WAGTAILFRONTENDCACHE = {}
+
 ENABLE_AKAMAI_CACHE_PURGE = os.environ.get('ENABLE_AKAMAI_CACHE_PURGE', False)
 if ENABLE_AKAMAI_CACHE_PURGE:
-    WAGTAILFRONTENDCACHE = {
-        'akamai': {
-            'BACKEND': 'v1.models.akamai_backend.AkamaiBackend',
-            'CLIENT_TOKEN': os.environ.get('AKAMAI_CLIENT_TOKEN'),
-            'CLIENT_SECRET': os.environ.get('AKAMAI_CLIENT_SECRET'),
-            'ACCESS_TOKEN': os.environ.get('AKAMAI_ACCESS_TOKEN')
-        },
+    WAGTAILFRONTENDCACHE['akamai'] = {
+        'BACKEND': 'v1.models.caching.AkamaiBackend',
+        'CLIENT_TOKEN': os.environ.get('AKAMAI_CLIENT_TOKEN'),
+        'CLIENT_SECRET': os.environ.get('AKAMAI_CLIENT_SECRET'),
+        'ACCESS_TOKEN': os.environ.get('AKAMAI_ACCESS_TOKEN')
     }
 
+ENABLE_CLOUDFRONT_CACHE_PURGE = os.environ.get('ENABLE_CLOUDFRONT_CACHE_PURGE', False)
+if ENABLE_CLOUDFRONT_CACHE_PURGE:
+    WAGTAILFRONTENDCACHE['cloudfront'] = {
+        'BACKEND': 'wagtail.contrib.wagtailfrontendcache.backends.CloudfrontBackend',
+        'DISTRIBUTION_ID': {
+            'files.consumerfinance.gov': os.environ.get('CLOUDFRONT_DISTRIBUTION_ID_FILES')
```

Comment:
Can this line (and the Akamai ones above) please be changed to use `os.environ['KEY']` instead of `os.environ.get('KEY')`? If the variable isn't in the environment, this should raise an exception, instead of passing by without error and sticking `None` in the dict.

Reply:
@chosak We already raise a value error if `AkamaiBackend` isn't initialized properly in `akamai_backend.py`.  It seems a bit overkill to prevent the entire site from running if it's misconfigured.

Reply:
Thanks @richaagarwal, I didn't realize we had that check there.